translationCoreApps/VerseCheck
# selectionHelpers to be tested

DO NOT MERGE!!!
This is for review purposes only.
The branch is continued to be worked on for following related tasks to prevent issues.

#### This pull request addresses:

Copied the functions from VerseCheck components to the helper directory so that tests can be run on them. During refactor, duplicates will be removed.

Slight refactoring of the code to ES6 and changes to pass params instead of using this.props like when it was in components.

DO NOT MERGE!!!
This is for review purposes only.
The branch is continued to be worked on for following related tasks to prevent issues.

#### How to test this pull request:

- Make sure the VerseCheck area still works. Note: there are pre-exising bugs.
- Review to see if I missed something related to Selections that should be extracted from the components and/or containers.

DO NOT MERGE!!!
This is for review purposes only.
The branch is continued to be worked on for following related tasks to prevent issues.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/versecheck/69)
